data/json/vitrification/furniture_vitrified.json

```
@@ -0,0 +1,874 @@
+    "move_cost_mod": -3,
+    "coverage": 50,
+    "required_str": 20,
+    "flags": [ "PLACE_ITEM", "BLOCKSDOOR", "MINEABLE" ],
+    "bash": {
+      "str_min": 16,
+      "str_max": 40,
+      "sound": "crunch!",
+      "sound_fail": "tink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_glassedbody_7",
+    "name": "black glass figure",
+    "description": "A black glass figure of someone clutching at their neck. A delicate spray of connected glass droplets emerges between their fingers.",
+    "symbol": "@",
+    "color": "dark_gray",
+    "move_cost_mod": -3,
+    "coverage": 50,
+    "required_str": 20,
+    "flags": [ "PLACE_ITEM", "BLOCKSDOOR", "MINEABLE" ],
+    "bash": {
+      "str_min": 16,
+      "str_max": 40,
+      "sound": "crunch!",
+      "sound_fail": "tink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_glassedbody_8",
+    "name": "black glass figure",
+    "description": "Most of a black glass human figure. The face and much of the head is missing, as if scooped out, and the inner surface is perfectly smooth.",
+    "symbol": "@",
+    "color": "dark_gray",
+    "move_cost_mod": -3,
+    "coverage": 50,
+    "required_str": 20,
+    "flags": [ "PLACE_ITEM", "BLOCKSDOOR", "MINEABLE" ],
+    "bash": {
+      "str_min": 16,
+      "str_max": 40,
+      "sound": "crunch!",
+      "sound_fail": "tink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_glassedbody_9",
+    "name": "black glass figure",
+    "description": "A black glass figure of someone doubled over in laughter.",
+    "symbol": "@",
+    "color": "dark_gray",
+    "move_cost_mod": -3,
+    "coverage": 50,
+    "required_str": 20,
+    "flags": [ "PLACE_ITEM", "BLOCKSDOOR", "MINEABLE" ],
+    "bash": {
+      "str_min": 16,
+      "str_max": 40,
+      "sound": "crunch!",
+      "sound_fail": "tink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_glassedbody_10",
+    "name": "black glass figure",
+    "description": "A black glass figure. Their features are too jagged to make out clearly, but they remind you of someone you knew.",
+    "symbol": "@",
+    "color": "dark_gray",
+    "move_cost_mod": -3,
+    "coverage": 50,
+    "required_str": 20,
+    "flags": [ "PLACE_ITEM", "BLOCKSDOOR", "MINEABLE" ],
+    "bash": {
+      "str_min": 16,
+      "str_max": 40,
+      "sound": "crunch!",
+      "sound_fail": "tink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_glassedbody_11",
+    "name": "black glass figure",
+    "description": "A black glass figure of a someone frozen halfway through falling backwards, and suspended on a single leg..",
```

```suggestion
    "description": "A black glass figure of someone in a fighting stance, their face twisted with howls of rage and anger.  They're gripping some sort of weapon that isn't there.",
```
This seems to be a duplicate of the first figure, so I came up with this for more variety.